### DISCUSSION TITLE
The repo mhutchie/vscode-git-graph seems stale.

### DISCUSSION THREAD

**Discussion by itaibh** [community/community#102993] [Repositories]
### Select Topic Area

General

### Body

The repo [vscode-git-graph](https://github.com/mhutchie/vscode-git-graph) seems stale as it has no new commits for over 2 years now, and no releases for over 3 years.
The [License](https://github.com/mhutchie/vscode-git-graph?tab=License-1-ov-file#readme) by "[mhutchie](https://github.com/mhutchie/)" (Michael Hutchison) as I understand it prevents publications of forks and derivative work, and from the community of developers no one picked it up from the moment Michael stopped commiting, perhaps they don't have write permissions.
Issues are piliing up, pull-requests also, but no one to handle them.
Is there a way for someone to take over the project? It's a great project which came to an abrupt stop before it could fulfill its full potential.
I already wrote an email to Michael, but no response.

**Comment by victor-huertas** [comment]
I realise your question is from two years ago but check out this fork if you haven't: https://github.com/asispts/neo-git-graph
